### PR TITLE
Add NLCD 2016 Edition Visual Layers

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Add the four new NLCD 2016 version visual layers to the Coverage Tab of the Layer Selector.

These were ingested via GDAL on EC2, using the script specified here: https://github.com/WikiWatershed/model-my-watershed/issues/3393#issuecomment-866179490

The old layer was renamed "National Land Cover Database Old 2011". /cc @darscott @SRGDamia1 @aufdenkampe for confirmation if this is appropriate, or if we should call it something else.

The layer selector was widened a bit to accommodate the lengthier layer names.

Connects #3394 

### Demo

The NLCD Analyze connects to the Old 2011 layer, this will be amended in #3395 

image

## Testing Instructions

* Check out this branch and `./scripts/bundle.sh --debug --vendor`
* Go to [:8000/](http://localhost:8000/)
* Switch the layer selector to the Coverage (second) tab
  - [x] Ensure you see 4 new NLCD layers, and they work correctly
  - [x] Ensure the old NLCD layer is renamed
* Choose a shape to analyze
  - [x] Ensure the Land Analysis still uses the Old 2011 layer, and it turns on from the sidebar correctly

### CODE REVIEW

File: src/mmw/mmw/settings/layer_settings.py [hunk 1]
```diff
@@ -90,12 +90,72 @@
     ],
     'coverage': [
         {
-            'display': 'National Land Cover Database',
+            'display': 'National Land Cover Database 2016',
+            'code': 'nlcd-2016-30m',
+            'css_class_prefix': 'nlcd-2016-30m nlcd',
```
Comment (rajadain) [inline src/mmw/mmw/settings/layer_settings.py:95]: The CSS class prefix has a qualifier for this specific layer `nlcd-2016-30m` but also the general `nlcd` class, which is used to fill in the background colors for the NLCD categories here:

https://github.com/WikiWatershed/model-my-watershed/blob/c7a6fcb82a08bb37c24c5fda479af9e37d328505/src/mmw/sass/utils/_variables.scss#L56-L79

**Comment by rajadain** [comment]
> I'm not sure if I fully tested this

Sorry, I should've been more explicit. But your screenshot does show it. Essentially, we want to ensure that the "Related Layer" says Old 2011 in the sidebar, and turning it on enables the correct Old 2011 layer:

image

(this is the same image as in the PR description, with red arrows added to highlight the changes)

**Comment by rajadain** [comment]
The CI is finally passing. Going to merge this now. Thanks for taking a look!

**Comment by darscott** [comment]
Terence,
can you replace the text on the Layer pallet to the following for the NLCD
layers?


   - Land Use/Cover 2016 (NLCD16)
   - Land Use/Cover 2011 (NLCD16)
   - Land Use/Cover 2006 (NLCD16)
   - Land Use/Cover 2001 (NLCD16)
   - Land Use/Cover 2011 (NLCD11)

___________________________________________________________________________________

Dave B. Arscott, Ph.D. - Executive Director, Research Scientist

Stroud Water Research Center -  Avondale, PA 19311 USA
Phone: (office) 610-268-2153 x 1278, (mobile) 484-885-5225

*Advancing knowledge and stewardship of fresh water through global
research, education, and restoration*
_____________________________________________________________________________________



*Making a gift/pledge in honor or memory of someone special is a
particularly meaningful way to aid the mission of the Stroud Water Research
Center Special Gifts program.  In addition, please consider a challenge
match grant by contacting me to establish match parameters. Make an online
donation now <https://stroudcenter.org/make-special-gift> or
call 610-268-2153, ext. 278, with any questions about contributing a gift
or a multi-year pledge. Established Challenge Match grants:*

   - *$1 Million Challenge Match Grant generously sponsored by Rod and
   Alice Moorhead to benefit the Bernard W. Sweeney, Ph.D., Executive
   Director’s Fund. * Your charitable donation will be matched
   dollar-for-dollar thanks to the generosity of Rod and Alice.
   - A $50,000 Challenge Match Grant by the Colket Family Foundation will
   match every gift to the Environmental Education Fund up to $50,000!

**Comment by rajadain** [comment]
Yes, I'll make that change. Thanks!

**Pull request merged**